### ISSUE TITLE
Alt + Fullscreen-Button

### ISSUE THREAD

**Issue by nerolapis** [weAreYeah/WAYWindow#14]
When alt-clicking on the fullscreen button (when the "+" sign appears in hover) the behaviour is not what is expected: the window should be zoomed but instead it enters in fullscreen mode.

To reproduce this issue just execute the test project and alt-click on the green traffic light button.

_Xcode 6.2 - OS X 10.10.2 - retina_

**Comment by steffanlevet** [comment]
Agree, defiantly a bug. I played with adding the subview at a different ordering position, without much luck. In the end I am able to get pretty good results by setting the frame to be offset by ~70px so it doesn't cover the traffic light buttons, and the default behaviour works again.
